metricbeat/module/mongodb/_meta/config.yml

```diff
@@ -1,9 +1,9 @@
-- module: mongodb
-  metricsets: ["status"]
-  enabled: true
-  period: 10s
+#- module: mongodb
+  #metricsets: ["status"]
+  #enabled: true
```

Comment:
I find the comments more visually clear. Also, this is inline with the rest of the modules.